Util/ComposerManager.php

```diff
@@ -1,11 +1,18 @@
 <?php
+declare(strict_types = 1);
 
 namespace Protacon\Bundle\TestToolsBundle\Util;
 
+use RuntimeException;
```

Comment:
Should also use file_put_contents and file_exists?

Reply:
This should be fixed now.